src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+server.port=8081
+## Spring DATASOURCE (DataSourceAutoConfiguration & DataSourceProperties)
+spring.datasource.url= jdbc:postgresql://localhost:5432/UserDB
+spring.datasource.username= admin
+spring.datasource.password= secret
```

Comment:
@MarkoLazovicCode Credentials should not be visible in our git repo. All configuration files containing sensitive data that are included in this conf. should be git ignored.